CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.0 (2019-08-14)
+
+- Added the ability for the summary action bar component to be declared inside the split view component. [#16](https://github.com/blackbaud/skyux-action-bars/pull/16)
+- Added support for @skyux-sdk/builder@3.8.1. [#15](https://github.com/blackbaud/skyux-action-bars/pull/15)
```

Comment:
I'm pretty sure our style is to make `@skyux-sdk/builder#3.8.1` a code snippet.

Reply:
Done